config.json

```diff
@@ -110,6 +110,12 @@
       ]
     },
     {
+      "slug": "bracket-push",
+      "difficulty": 3,
```

Comment:
OK.

The only other exercise I can think of that would get `Stack` in its topic list at the same time would be `forth`.